src/Lib/Managers/NobManager.vala

```diff
@@ -350,7 +350,9 @@ public class Akira.Lib.Managers.NobManager : Object {
         if (selected_items.length () == 1) {
             var item = selected_items.nth_data (0);
 
-            item.get_transform (out transform);
+            transform = item.get_real_transform ();
+
+            // item.get ("line_width", out line_width);
```

Comment:
Do we need this?

Reply:
Yes, as per the answer I gave to the other comment related to the SelectedBoundManager.ù

We now need to take into consideration whether the item is inside an Artboard or nor. 
In the latter, the transformation does involve using the parent's coordinate system to calculate the proper transform to apply.